File: test/resources/HelloProphecy/prophecy/pipelines/silver_zip_codes/code/silver_zip_codes/graph/bronze_irs_zipcode.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from silver_zip_codes.config.ConfigStore import *
from silver_zip_codes.udfs.UDFs import *

def bronze_irs_zipcode(spark: SparkSession) -> DataFrame:
    return spark.read.table("`scottdemo`.`bronze_irs_zipcode`")

File: test/resources/HelloProphecy/prophecy_scala/pipelines/silver_zip_codes/code/src/main/scala/io/prophecy/pipelines/silver_zip_codes/graph/bronze_irs_zipcode.scala
package io.prophecy.pipelines.silver_zip_codes.graph

import io.prophecy.libs._
import io.prophecy.pipelines.silver_zip_codes.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object bronze_irs_zipcode {

  def apply(context: Context): DataFrame =
    context.spark.read.table("`scottdemoscala`.`bronze_irs_zipcode`")

}
